projects/sunbird-pdf-player/src/lib/playerInterfaces.ts

```diff
@@ -25,15 +25,15 @@ export interface ObjectRollup {
 }
 
 export interface Context {
-    mode: string;
+    mode?: string;
```

Comment:
If all these properties are optional, then why "context" is mandatory in the "PlayerConfig"?

Reply:
If you make all are optional, How the telemetry events will generate if any required property is missing for telemetry like did, uid etc..